test/test_send.f90

```diff
@@ -1,19 +1,21 @@
-program name
-  use redis_mod, only: array_to_redis
-  implicit none
-  integer, parameter ::n=200, m=100
-  integer :: dims(2)
-  integer f(n,m)
-  integer ndims
-  integer i, j
-  do j=1,m
-    do i=1,n
-      f(i,j) = i
-    end do
-  end do
+PROGRAM name
+  USE redis_mod, ONLY: array_to_redis
+  IMPLICIT NONE
+
+  INTEGER, PARAMETER ::n=200, m=100
+  INTEGER            :: dims(2)
+  INTEGER            :: f(n,m)
+  INTEGER            :: ndims
+  INTEGER            :: i, j
+
+  DO j=1,m
+     DO i=1,n
+        f(i,j) = i
+     END DO
+  END DO
 
   ndims = 2
   dims = (/n,m/)
 
-  call array_to_redis(f)
-end program name
+  CALL array_to_redis(f)
```

Comment:
I'm not really clear on what the purpose of this function is.

Reply:
I think this is the function we used to serve data to the bokeh server for the demonstration.

Reply:
okay, I've removed this then.